gridstatus/decorators.py

```diff
@@ -330,3 +330,28 @@ def pjm_update_dates(dates, args_dict):
             )
 
     return new_dates
+
+
+def ercot_update_dates(dates, args_dict):
+    date = args_dict["date"]
+    end = args_dict["end"]
+    years = {x for x in range(date.year, end.year + 1)}
+
+    fixed_dates = []
+
+    for i, year in enumerate(years):
```

Comment:
I'm not sure this logic is right.

what happens if you request data like

start = "Feb 1, 2022"
end = "March 2, 2022"

I think in this case something will go wrong.

Reply:
if not, can you add a test showing it works?